src/_N001_Email.ino

```diff
@@ -108,10 +110,24 @@ boolean NPlugin_001_send(const NotificationSettingsStruct& notificationsettings,
 			mailheader.replace(String(F("$emailfrom")), address);
 		}
 
-		mailheader.replace(String(F("$nodename")), Settings.Name);
-		mailheader.replace(String(F("$emailfrom")), notificationsettings.Sender);
+		//mailheader.replace(String(F("$nodename")), Settings.Name);
+		//mailheader.replace(String(F("$emailfrom")), notificationsettings.Sender);
 		mailheader.replace(String(F("$ato")), notificationsettings.Receiver);
-		mailheader.replace(String(F("$subject")), aSub);
+
+		int sub_left = aMesg.indexOf('{');
+		int sub_right = aMesg.indexOf('}');
+		if (sub_left == -1 || sub_right == -1) {
+		    mailheader.replace(String(F("$subject")), aSub);
```

Comment:
No need to wrap the `F()` macro in a `String()`, Arduino knows how to take care of that `F()`, and this will cost extra code and memory.